.travis.yml

```diff
@@ -77,6 +77,7 @@ script:
   - go build -i .
   - make check
   - if [ "$GOOS" = "linux" ]; then make check-protos check-api-descriptors; fi
+  - if [ "$TRAVIS_GOOS" = "linux" ]; then make man ; fi
```

Comment:
what's the difference between GOOS and TRAVIS_GOOS, if both are used from .travis.yml?

Reply:
Yes, we should probably check what the differences are (if any) and clean up (if possible); either put a comment why use one over the other, or remove one of both (because it's definitely confusing)

Reply:
`TRAVIS_GOOS` is just defined in the env section and used in this yml, it has no other meaning. Not sure why `GOOS` is used above, but `TRAVIS_GOOS` should be used here since we set it and know what it is set to.